nbresuse/__init__.py

```diff
@@ -1,100 +1,17 @@
 import os
-import json
-import psutil
+
+from tornado import ioloop
 from traitlets import Bool, Float, Int, Union, default
 from traitlets.config import Configurable
-from notebook.utils import url_path_join
-from notebook.base.handlers import IPythonHandler
-from tornado import web
+
+from nbresuse.prometheus import PrometheusHandler
 
 try:
     # Traitlets >= 4.3.3
     from traitlets import Callable
 except ImportError:
     from .utils import Callable
 
-from concurrent.futures import ThreadPoolExecutor
-from tornado.concurrent import run_on_executor
-
```

Comment:
Prometheus client already is a dependency of the notebook: https://github.com/jupyter/notebook/blob/master/setup.py#L114 so i am not sure about the supporting of prometheus_client optional. It would make it possible to use the extension for some fairly old versions of notebook (2+ years or so?).

Reply:
https://github.com/jupyter/notebook/commit/a764f90b142df9eb4bb4d99dc610a83623f1f390
Added in 5.6.0, released 5 Nov 2018 https://github.com/jupyter/notebook/releases/tag/5.6.0

Reply:
oh, so only a year ago, so that might be reason enough to keep a legacy version?

Reply:
I think it would also be reasonable to say this requires `notebook>=5.6`, and make it clear that older releases of nbresuse will still work. Are there any significant new features that aren't available in the existing release?

Reply:
@krinsman what is your opinion on this?

Reply:
Oh, I see your point @Tommassino -- and anything that is a dependency of notebook is also a dependency of JupyterLab, because JupyterLab depends on notebook.

For some reason I thought notebook and JupyterLab had different dependencies (i.e. the latter wouldn't depend on prometheus-client)? 

I was worried about making prometheus a dependency when it was a dependency for notebook but not JupyterLab, but as doing an installation of JupyterLab via conda shows, prometheus_client is very much a dependency of both, for the reason stated above.

OK, yes, in this case, I agree that the `prometheus_client` dependency probably doesn't need to be optional then. I think @manics is correct, we could just add `notebook>=5.6` to the `requirements.txt` of newer versions of `nbresuse`, and this shouldn't bite anyone. Especially since it won't retroactively add `prometheus_client` as a dependency to older versions of `nbresuse` which people should hypothetically still be able to install via pip.

@Tommassino, OK, feel free to disregard everything I said about the `prometheus_client` thing, it was me being stupid.